contents/projectManagement/githubActions.md

```diff
@@ -0,0 +1,143 @@
+
+* **Action**
+    An action is a piece of code that performs an atomic task. 
+    For example, [checking out a repository](https://github.com/actions/checkout), or [caching dependencies](https://github.com/actions/cache).
+
+* **Triggers**
+    Workflows can be triggered by events on the GitHub repository, ranging from opening pull requests or issues, making commits to scheduled events, or even from an external event by calling GitHub's REST API endpoint.
+
+---
+
+## Set Up a Simple Workflow for a Node.js App
+<!--
+Provide a learning path for the reader. Try to give one good learning path rather than many random resources.
+--> 
+
+GitHub has made it extremely easy to set up a simple workflow without even leaving the browser. 
+
+![](images/actions-tab.png)
+
+1. Head to the Actions tab on your repository
+1. You can choose to set up a workflow from a template or from scratch. We will use the Node.js template in this example
+1. The online editor is powerful and easy to use, boasting features such as inline documentation and autocomplete
+![](images/editor.png)
+1. The template workflow (same as the example above) will
+    1. Begin when a commit is pushed or a pull request is opened to `master`
+    1. Checkout the current repo (using `actions/checkout`)
+    1. Install dependencies `npm ci`
+    1. Build `npm run build --if-present`
+    1. Run tests `npm run test`
+    1. Report the status of the workflow via a status badge
+    ![](images/status.png)
+1. Since we set the workflow up on the `master` branch, when we commit the newly-created workflow file, we can see GitHub Actions in action by navigating to the Actions tab
+    ![](images/progress.png)
+
+---
+
+## Where to Go From Here
+<!--
+Give more resources. Instead of listing a lot of links, provide a brief summary of what value each resource can provide the reader.
+-->
+The official [documentation](https://help.github.com/en/actions) is a good resource to learn more about GitHub Actions.
+The [starter workflows](https://github.com/actions/starter-workflows) repository contains good examples of 
+CI workflows for popular languages as well as automation workflows for GitHub.
+Here are a list of [awesome actions](https://github.com/sdras/awesome-actions) that you can use when writing your workflow.
+ 
+
+A powerful extension to GitHub Actions is to use the [github-script](https://github.com/actions/github-script) action 
+to run JavaScript within the workflow using GitHub's [Octokit](https://octokit.github.io/rest.js/v17/) API. 
+The Octokit API allows you to create workflows that span across GitHub's ecosystem of services.
+
+I have also compiled a list of popular repositories that use GitHub Actions that you can use as a reference. Do note that this is far from a comprehensive list and is more of a sample of how organizations are leveraging GitHub Actions in their development and production workflow.
+
+<panel header="Repositories that use GitHub Actions">
```

Comment:
This panel title can be more aligned with the purpose. e.g., `Examples of different ways GitHub Actions can be used`